### ISSUE TITLE
Detailed instructions for proper installation and use (not only for MultiMC as i guess)

### ISSUE THREAD

**Issue by Van-Buren** [THEREALWWEFAN231/Inertia#1332]
**Minecraft Version(s):** 1.16.5 (MultiMC launcher)

**What is the suggestion:** Specify the required versions of the libraries on the download/installation page of Inertia (in this case the version of Fabric and Java)

**Why would this feature be useful:** This will make it easier to install and use the client, and people will stop (hopefully) creating new issues here and on other resources related to problems that are solved on other resources (in this case, on Discord). Since in this particular case, all these requests are a clogging of the air and they delay the correction of other unresolved problems that require correction.

**What's happened?:** I wanted to install Inertia client on Minecraft when I read reviews about his Notebot (which claimed that he was one of the best) and in addition there is a convenient ViaVersion function. After installing the most current version available (Minecraft 1.16.5 client version 3.1.3) in the MultiMC launcher (the most convenient and functional for me), problems started at the first start. After launching Minecraft and going to the multiplayer menu, I found the lack of a ViaVersion button. My first thought was to try installing this mod separately. Sometimes it worked, but more often it caused errors and the game did not want to start. After checking the video with Inertia reviews, I found that on this version the ViaVersion function should be present, which means that something is wrong with my game setup. After trying many different versions and variations of a ViaVersion-like mod, i tried to find error messages related to same launcher (MultiMC) from other people and came across a post on reddit where another user had a problem with launching the game. In response to this message, you (THEREALWWEFAN231) recommended that the user go to your Discord server and check the pinned messages in the general channel. I decided to do the same, and how surprised I was to see that the user Godeps (I don’t know exactly who this is and whether he is somehow connected with the installation information and the FAQ posted on the Inertia site, except for what he has on this server there is a "Support" role) left such a useful tip in which he recommended using Fabric version 0.10.4 or older. After I installed this version, and also chose the Java 8 version, Minecraft started normally and the coveted viaversion button appeared in the multiplayer menu. What did I mean by this? If your project is aimed at making people's lives easier (as I understand it), then why did I have to spend so much time to get some simple recommendations on how to install and fix fairly common errors? All I ask is to post information about the need to use Java 8 and Fabric 0.10.4 for Minecraft versions 1.16.5 and 1.15.2 (I did not check other versions except 1.12.2, but such problems should not have arisen here due to Forge instead of Fabric). I did not have the opportunity to check the performance on the official launcher, because. it doesn't work on my current (temporary) device. Thank you for your attention.
PS: I apologize for the use of machine translation, I hope this does not make it difficult to understand all of the above.

**Final checklist:**

- [x] I know how to properly use check boxes 
- [x] This is not a suggestion for baritone or other mod (i guess)

**Comment by Realswitzer** [comment]
![image](https://user-images.githubusercontent.com/39711351/221310790-cc08ed4d-8866-478e-b686-2635d31c137d.png)
![image](https://user-images.githubusercontent.com/39711351/221311169-1905faf4-0960-4667-b0e1-9e0429bbfaf1.png)

inertia has not been updated in quite a while, and nobody knows why viaversion went poof.
plus tbh, at least in #general, nobody asks for help and likely until any update, will continue to not have help requests in general. don't have support role so can't check tickets.

tldr of original post: no viaversion button, who is godeps, used fabric 0.10.4 and java 8 and it worked. (sorry if anything major is missing from this tldr.)

**Comment by Van-Buren** [comment]
> ![image](https://user-images.githubusercontent.com/39711351/221310790-cc08ed4d-8866-478e-b686-2635d31c137d.png) ![image](https://user-images.githubusercontent.com/39711351/221311169-1905faf4-0960-4667-b0e1-9e0429bbfaf1.png)
> 
> inertia has not been updated in quite a while, and nobody knows why viaversion went poof. plus tbh, at least in #general, nobody asks for help and likely until any update, will continue to not have help requests in general. don't have support role so can't check tickets.
> 
> tldr of original post: no viaversion button, who is godeps, used fabric 0.10.4 and java 8 and it worked. (sorry if anything major is missing from this tldr.)

Thanks for your reply. Actually, I don't care who Godeps is, I just mentioned him as a user whose advice helped me (and I guess will help other users who have the same problems) and who, as I understand it, can somehow contact the developer of Inertia or his site. In addition to the results that you summed up, I will repeat that, in my opinion, it is worth mentioning on the Inertia website about the need for certain versions of the Fabric and Java (at least when using MultiMC).
